content/guides/core/automations/automation-events.md

```diff
@@ -45,10 +45,46 @@ You can create a project automation at these scopes:
 - Project level: The automation watches for the event taking place on any collection in the project.
 - Collection level: All collections in the project that match the filter you specify.
 
-### Events
-A project automation can watch for these events:
-- **A new version of an artifact is created in a collection**: Apply recurring actions to each version of an artifact. Specifying a collection is optional. For example, start a training job when a new dataset artifact version is created.
-- **An artifact alias is added**: Trigger a specific step of your workflow when a new artifact version in a project or collection has a specific alias applied. For example, run a series of downstream processing steps when an artifact has the `test-set-quality-check` alias applied.
+### Artifact events
+This section describes the events related to an artifact that can trigger an automation.
+
+- **Linking a new artifact**: Test and validate new models, datasets or dataset automatically.
+- **Creating a new version of an artifact**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created.
+- **Adding a new alias to a version of an artifact**: Trigger a special step of your workflow when a new artifact version in a project or collection has a specific label or alias applied. For example, run a series of downstream processing steps when an artifact has the `test-set-quality-check` alias applied.
+
+### Run metrics events
+An automation can watch for changes in a run metric:
+
+- **Run metrics threshold met**: Trigger a workflow when for a given metric, a single logged value or the average logged values meets the threshold you specify.
+- **Run metrics change threshold met**: Trigger a workflow when for a given metric, a single logged value or the average logged value changes by the threshold you specify.
+
+Each run is considered separately, even if the automation has a run filter that matches with multiple runs. Each run's values are put into a separate window and compared to the threshold separately. If an automation starts due to the result of one run, it can still start for another matching run. A particular run metric automation runs only once per run per 24-hour period.
+
+{{% alert %}}
+You can create automations triggered by system metrics such as the `cpu` metric, which tracks the percentage of CPU utilization. W&B logs system metrics automatically every 15 seconds. Find each metric's name and details in [System metrics]({{< relref "/guides/models/app/settings-page/system-metrics.md" >}}).
```

Comment:
ditto that this feels random put in here - I don't see why it needs to have an alert. If we created a 'metric name' section, we could put this in there

This would be fine to do in a follow on change since I know you're under time pressure.